dataplug/app/com/hubofallthings/dataplug/actors/DataPlugManagerActor.scala

```diff
@@ -176,7 +177,10 @@ class DataPlugManagerActor @Inject() (
 
           eventualSyncMessage.recover {
             case e =>
-              logger.error(s"Error while trying to start syncing dat for $phata, variant $variant: ${e.getMessage}", e)
+              logger.error(s"Error while trying to start syncing data for $phata, variant $variant: ${e.getMessage}", e)
+              val sw = new StringWriter
```

Comment:
Non Blocking: Is this the best way to print a stack trace ? Ive never seen string writer utilised for this

Reply:
I didn't know how to print the stack trace as I haven't done it before, so I googled. 
https://alvinalexander.com/scala/how-convert-stack-trace-exception-string-print-logger-logging-log4j-slf4j/

I think Alvin Alexander is a well-known person in Scala world so I didn't research further. If it's not ideal, or you know another way of doing it feel free to share!!! I will learn something new!